spring-cloud-aws-messaging/src/main/java/org/springframework/cloud/aws/messaging/listener/SimpleMessageListenerContainer.java

```diff
@@ -440,7 +442,23 @@ private void applyDeletionPolicyOnError(String receiptHandle) {
 
 		private void deleteMessage(String receiptHandle) {
 			getAmazonSqs().deleteMessageAsync(
-					new DeleteMessageRequest(this.queueUrl, receiptHandle));
+					new DeleteMessageRequest(this.queueUrl, receiptHandle),
```

Comment:
Since this handler is exactly the same as in previous class (for a good reason), wouldn't it make sense to extract it to an actual class instead of anonymous class?

Reply:
Done